corehq/messaging/smsbackends/http/sms_sending.py

```diff
@@ -0,0 +1,21 @@
+from corehq.util.urlvalidate.urlvalidate import (
+    PossibleSSRFAttempt,
+    validate_user_input_url,
+)
+
+from corehq.apps.sms.models import SMSBase
+
+from corehq.util.metrics import metrics_counter
+
+
+def verify_sms_url(url, msg, backend):
```

Comment:
Possibly. I'm trying to avoid 'util' as it doesn't really convey any meaning and easily turns into a generic catch-all file. Given that very few of our backends actually support URLs, I'm not sure it makes more sense to put this into _smsbackends_ or _messaging_ rather than just leaving it scoped to the code most tightly coupled to it. I could see adjusting the name `sms_sending` to something more URL specific, such as `url_validation.py`. Any strong thoughts against leaving the structure as-is?

Reply:
Nope, that's fine.